Vivliostyle CLI 8.1.0 fails to run

```
$ vivliostyle --version
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/user/.local/lib/node_modules/@vivliostyle/cli/vendors/index.js' imported from /home/user/.local/lib/node_modules/@vivliostyle/cli/dist/const.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:324:11)
    at moduleResolve (node:internal/modules/esm/resolve:943:10)
    at defaultResolve (node:internal/modules/esm/resolve:1129:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.17.0

$ ls ~/.local/lib/node_modules/@vivliostyle/cli/vendors/
README.md  build.js  index.d.ts  index.src.js
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,5 @@
 /dist
+/vendors/index.js
 /tmp
 *.pdf
 .vscode

--- a/vendors/.gitignore
+++ b/vendors/.gitignore
@@ -1,1 +1,0 @@
-index.js

